app/controllers/spotlight/compounds_controller.rb

```diff
@@ -0,0 +1,86 @@
+module Spotlight
+	class CompoundsController < Spotlight::ApplicationController
+		include Blacklight::SearchHelper
+		load_and_authorize_resource :exhibit, class: Spotlight::Exhibit, prepend: true
+		before_action :authenticate_user!, only: [:new, :create, :update, :edit]
+		before_action :check_authorization, only: [:new, :create, :update, :edit]
+		require 'fileutils'
+		
+		def index
+			@exhibit = Spotlight::Exhibit.find params[:exhibit_id]
+			@resources = Spotlight::Resource.where("data like '%items%' and exhibit_id = '#{@exhibit.id}'")
```

Comment:
This raw query is surprising. If we need to handle these differently, maybe a subclass of `Spotlight::Resource` (like we're doing for the uploaded resources) would be more appropriate?